Bottom: 6c2adfbba059f014aacd7846d6d74ba9a8b7a9fe
Top:    db1711d77333c4af49851bef860eb92c73f5f70f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-30 15:52:35 +0800

Refresh of vhost_net-split-out-datacopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index a868a92..33e6428 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -626,12 +626,9 @@ static void handle_tx_zerocopy(struct vhost_net *net)
 	vhost_disable_notify(&net->dev, vq);
 	vhost_net_disable_vq(net, vq);
 
-	zcopy = nvq->ubufs;
-
 	for (;;) {
 		/* Release DMAs done buffers first */
-		if (zcopy)
-			vhost_zerocopy_signal_used(net, vq);
+		vhost_zerocopy_signal_used(net, vq);
 
 		err = get_tx_bufs(net, nvq, &used, &msg, &out, &in, &len);
 		if (err == -ENOSPC) {
